### DISCUSSION TITLE
Umi qiankun通信问题

### DISCUSSION THREAD

**Discussion by consistent-k** [umijs/umi#12008] [Q&A]
## What happens?
使用官方这段代码进行父子应用通信，求问在父应用其他页面中，如何能调用这里的函数 将异步请求的数据设置给子应用。

## Mini Showcase Repository(REQUIRED)
```js
// src/app.ts
export function useQiankunStateForSlave() {
  const [globalState, setGlobalState] = useState<any>({
    slogan: 'Hello MicroFrontend',
  });

  return {
    globalState,
    setGlobalState,
  };
}
```
不明白如何再页面内调用这段。

## How To Reproduce

按照umi文档操作

## Context
- **Umi Version**: 4.0.90
- **Node Version**：18
- **Platform**: macos

**Comment by fz6m** [comment]
See #12006
